src/java/org/apache/cassandra/db/compaction/CompactionStrategyOptions.java

```diff
@@ -57,7 +57,7 @@
     public static final String DEFAULT_UNCHECKED_TOMBSTONE_COMPACTION_OPTION = "false";
     public static final String DEFAULT_LOG_TYPE_OPTION = System.getProperty("default.compaction.logs", "none");
     public static final String DEFAULT_LOG_PERIOD_MINUTES_OPTION = System.getProperty("default.compaction.log_minutes", "1");
-    public static final String DEFAULT_READ_MULTIPLIER_OPTION = System.getProperty("default.compaction.costs_read_multiplier", "0.5");
+    public static final String DEFAULT_READ_MULTIPLIER_OPTION = System.getProperty("default.compaction.costs_read_multiplier", "1.0");
```

Comment:
This should change does not belong here, because its relationship with the other changes in this PR will be confusing for someone looking at the blame output.

You can open another pull request for STAR-1864 and apply it there.